### PR TITLE
GUI for manual addition of comm frequencies for satellites

### PR DESCRIPTION
**Pull request by alex-w**
This feature was added into TODO list (item: _GUI for manual addition of comm frequences_) for Satellites plugin at least 9 years ago, but it was probably earlier. I've added the GUI to adding and removing the communication data for individual satellites only - the tool doesn't support groups of satellites (this is intended behaviour!).

### Screenshots:
satellites_plugin_satellites
satellites_plugin_comms

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (header file)
- [x] I have updated the respective chapter in the Stellarium User Guide
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by alex-w** [comment]
OK, it's ready for merge

File: plugins/Satellites/src/gui/SatellitesCommDialog.cpp [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+			c.modulation = modulation;
+
+		communications.append(c);
+		SatellitesMgr->getById(satelliteID)->setCommData(communications);
+		SatellitesMgr->saveCatalog();
+
+		getSatCommData();
+	}
+	else
+		QMessageBox::warning(&StelMainView::getInstance(), q_("Warning!"), q_("Please fill description and set the frequency!"), QMessageBox::Ok);
```
Comment (gzotti) [inline plugins/Satellites/src/gui/SatellitesCommDialog.cpp:195]: I think we should give a less formal message than "Warning". Maybe:

Nothing added / Please fill in description and add frequency before adding!

Comment (alex-w) [reply plugins/Satellites/src/gui/SatellitesCommDialog.cpp:195]: Good idea, thanks!

**Comment by alex-w** [comment]
Done.

What about "Data description" as suggestion for field 'description' and "Communications" as title?

**Comment by gzotti** [comment]
> Done.
> 
> What about "Data description" as suggestion for field 'description' and "Communications" as title?

Probably: 

Communications --> Frequencies

Data Description: I don't know, it may be ok, or be something like "channel name" or "link name"? Maybe Jay can propose a better name. (What was his user name?)

**Comment by alex-w** [comment]
> Communications --> Frequencies

Good suggestion.

> Data Description: I don't know, it may be ok, or be something like "channel name" or "link name"? Maybe Jay can propose a better name. (What was his user name?)

Channel name looks good. 

@JAY-Githb could you review it too?

**Comment by alex-w** [comment]
@gzotti what about changes?

**Comment by gzotti** [comment]
I thought we wait for a suggestion by @JAY-Githb. Else OK for me.

**Comment by alex-w** [comment]
👍 

Thanks!

**Comment by JAY-Githb** [comment]
Catching up on email. 
I just replied to Alex that my work is visual, not radio, so I can't make suggestions for communications. 
Sorry. 

Jay Respler 
Advanced Business Machines Co. 
609-662-7438 
***@***.*** 
Monroe Township, New Jersey

**Pull request merged**